clients/discovery.go

```diff
@@ -0,0 +1,55 @@
+package clients
+
+import (
+	"fmt"
+	"terraform-provider-ome/models"
+)
+
+func (c *Client) CreateDiscoveryJob(discoveryJob models.DiscoveryJobPayload) (models.DiscoveryJob, error) {
+	data, _ := c.JSONMarshal(discoveryJob)
+	response, err := c.Post(DiscoveryJobAPI, nil, data)
+	if err != nil {
+		return models.DiscoveryJob{}, err
+	}
+	respData, _ := c.GetBodyData(response.Body)
+	omeDiscoveryJob := models.DiscoveryJob{}
+	err = c.JSONUnMarshal(respData, &omeDiscoveryJob)
+	return omeDiscoveryJob, err
+}
+
+func (c *Client) UpdateDiscoveryJob(discoveryJob models.DiscoveryJobPayload) (models.DiscoveryJob, error) {
+	data, _ := c.JSONMarshal(discoveryJob)
+	x := DiscoveryJobAPI + "?groupId=" + fmt.Sprint(discoveryJob.DiscoveryConfigGroupID)
+	response, err := c.Post(x, nil, data)
```

Comment:
Please try to use the Post method's query params attribute 
```suggestion
	response, err := c.Post(DiscoveryJobAPI, map[string]string{
	        "groupId": discoveryJob.DiscoveryConfigGroupID,
	}, data)
```

Reply:
add to group id to post method query params